src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { memo } from 'react';
+import cn from 'classnames';
+import { FilterType } from '../../types/FiltersType';
+
+type Props = {
+  activeTodos: number;
+  completedFilter: FilterType;
+  onChangeType: (str: FilterType) => void;
+  completedTodos: number;
+  onDeleteComplited: () => void;
+};
+
+export const Footer: React.FC<Props> = memo(({
+  activeTodos,
+  completedFilter,
+  onChangeType,
+  completedTodos,
+  onDeleteComplited,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${activeTodos} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={cn('filter__link', {
+            selected: completedFilter === FilterType.all,
+          })}
+          onClick={() => onChangeType(FilterType.all)}
+        >
+          All
+        </a>
+
+        <a
+          data-cy="FilterLinkActive"
+          href="#/active"
+          className={cn('filter__link', {
+            selected: completedFilter === FilterType.active,
+          })}
+          onClick={() => onChangeType(FilterType.active)}
+        >
+          Active
+        </a>
+        <a
+          data-cy="FilterLinkCompleted"
+          href="#/completed"
+          className={cn('filter__link', {
+            selected: completedFilter === FilterType.completed,
+          })}
+          onClick={() => onChangeType(FilterType.completed)}
+        >
+          Completed
+        </a>
+      </nav>
+
+      <button
+        data-cy="ClearCompletedButton"
+        type="button"
+        className="todoapp__clear-completed"
```

Comment:
also try to find how to hide your button using classes (bulma) or using inline styles